web/modules/custom/dpl_go/dpl_go.deploy.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/**
+ * Make sure that we have properly configured the Next Site.
+ */
+function dpl_go_deploy_0001_create_next_go_site_configuration(): string {
+  if (
+    getenv('LAGOON_ENVIRONMENT_TYPE') === 'production' &&
```

Comment:
Should we collect all the "sniff out lagoon env var" in a `dpl_lagoon` service?

Reply:
Hmm. Have to think about that. I'll go with this for now.